app/controllers/APIController.scala

```diff
@@ -0,0 +1,39 @@
+package controllers
+
+import java.time.LocalDateTime
+import javax.inject._
+import play.api._
+import play.api.mvc._
+import play.api.libs.json._
+import play.api.Environment._
+
+import java.time.format.DateTimeFormatter
+
+/**
+ * This controller creates an `Action` to handle HTTP requests to the
+ * application's home page.
+ */
+@Singleton
+class APIController @Inject()(val controllerComponents: ControllerComponents) extends BaseController {
+
+  /*
+   * method that returns a json object with the following information:
+   * { "service" : "pass-bakery"
+   *   "environment" : string (production or development run, based on play fw env config)
+   *   "serverTime" : string (local time from dateTime library in ISO 8601 format)
+   * */
+  def getStatus() = Action { implicit request: Request[AnyContent] =>
+    
+    play.api.Mode.Prod
+
+    // construct the json
```

Comment:
Comments should tell you the why, not the how something is done. I'm able to read that you are declaring a json here, but don't have any insight as to why you are doing that

Reply:
removed the comment